tests/js/wait/index.js

```diff
@@ -0,0 +1,110 @@
+/*
+
+    Test the "wait" function.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+require("../../../");
+gpii.webdriver.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.webdriver.wait");
+
+gpii.tests.webdriver.wait.displayAlert = function () {
+    setTimeout(function () { window.alert("Danger! Intruders among us!"); }, 1000);
+};
+
+gpii.tests.webdriver.wait.confirmTimeoutMessage = function (message, expectedText) {
+    // I have to require this here or the whole suite will time out with "no tests found" errors.
+    // TODO: Review and discuss with Antranig.
```

Comment:
I removed the workaround and required jqUnit from the top of the file.  The problem is no longer there, so I assume that a) it was the 13ms timeout we were hitting and b) the first for [FLUID-5810](https://issues.fluidproject.org/browse/FLUID-5810) is enough for us.